package.json

```diff
@@ -13,6 +13,7 @@
     "url": "https://github.com/balance-io/balance-manager/issues"
   },
   "dependencies": {
+    "@0x/instant": "^1.0.1",
```

Comment:
You don't need this :D 

We decided not to support integrating through NPM for now because it allows us to push upgrades / fixes out more easily. 

For example, we figured out the history bug, and will push it out ASAP. See here for more info https://github.com/0xProject/0x-monorepo/pull/1362